ecdsa/signing/messages.go

```diff
@@ -20,126 +20,111 @@ import (
 
 // These messages were generated from Protocol Buffers definitions into ecdsa-signing.pb.go
 
-var (
-	// Ensure that signing messages implement ValidateBasic
-	_ = []tss.MessageContent{
-		(*SignRound1Message1)(nil),
-		(*SignRound1Message2)(nil),
-		(*SignRound2Message)(nil),
-		(*SignRound3Message)(nil),
-		(*SignRound4Message)(nil),
-		(*SignRound5Message)(nil),
-		(*SignRound6Message)(nil),
-		(*SignRound7Message)(nil),
-	}
-)
+// Ensure that signing messages implement ValidateBasic
+var _ = []tss.MessageContent{(*SignRound1Message)(nil), (*SignRound2Message)(nil), (*SignRound3Message)(nil), (*SignRound4Message)(nil), (*SignRound5Message)(nil), (*SignRound6Message)(nil), (*SignRound7Message)(nil)}
 
 // ----- //
 
-func NewSignRound1Message1(
-	to, from *tss.PartyID,
+func NewSignRound1Message(
+	from *tss.PartyID,
 	c *big.Int,
-	proof *mta.RangeProofAlice,
+	proof *mta.RangeProofAlice, commitment cmt.HashCommitment,
 ) tss.ParsedMessage {
 	meta := tss.MessageRouting{
 		From:        from,
-		To:          []*tss.PartyID{to},
-		IsBroadcast: false,
+		IsBroadcast: true,
 	}
-	pfBz := proof.Bytes()
-	content := &SignRound1Message1{
+
+	out := proof.Bytes()
+	content := &SignRound1Message{
 		C:               c.Bytes(),
-		RangeProofAlice: pfBz[:],
+		RangeProofAlice: out[:],
+		Commitment:      commitment.Bytes(),
 	}
 	msg := tss.NewMessageWrapper(meta, content)
 	return tss.NewMessage(meta, content, msg)
 }
 
-func (m *SignRound1Message1) ValidateBasic() bool {
+func (m *SignRound1Message) ValidateBasic() bool {
 	return m != nil &&
 		common.NonEmptyBytes(m.GetC()) &&
 		common.NonEmptyMultiBytes(m.GetRangeProofAlice(), mta.RangeProofAliceBytesParts)
```

Comment:
The check from SignRound1Message2 did not carry over
```go
		m.Commitment != nil &&
		common.NonEmptyBytes(m.GetCommitment())
```

Reply:
Thank you, updated the code to include the commitment check!